XCSnippetr/Class/ViewControllers/XCSMainWindowController.m

```diff
@@ -418,6 +418,13 @@ - (void)didSubmitSnippetWithError:(NSError *)error
 {
     if (!error) {
         [[NSSound soundNamed:kSystemSoundSuccess] play];
+        
+        if (self.snippet.url) {
+            NSPasteboard *pasteboard = [NSPasteboard generalPasteboard];
+            [pasteboard clearContents];
+            [pasteboard writeObjects:@[ self.snippet.url ]];
```

Comment:
NSURL would not initialize with garbage, it returns nil or throws exception IIRC.